charters/DataOperations/charter.md

```diff
@@ -0,0 +1,102 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team (DataOps) is a user-focused group aiming to connect the scientific community to the DCP through the effective tools and resources generated by the DCP which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution. These products include data, metadata, analysis tools, data access features. DataOps ensures that data flows through the DCP end-to-end in an efficient manner, is appropriately validated, and fulfills the needs and expectations of data consumers. DataOps maintains a DCP-wide vision through their internal focus on end-to-end data flow, as well as their community-focused goal of conveying the value of the entire DCP.
+
+## Definitions
+
+*Data contribution*: a set of raw data derived from the same experimental process, and the metadata describing that process
+
+*Release* (verb): to make metadata and data openly accessible
+
+*Data distribution*: the packaging of metadata and data from multiple data contributions in meaningful ways, along with relevant analysis pipelines, possible integrative analysis results, user vignettes, access features, and any announcements or documentation supporting it
+
+*HCA Snapshot*: a type of data distribution that occurs on a regular cadence containing all HCA data currently available that meets predetermined criteria, such as quality standards
+
+*Data lifecycle*: the sequence of tasks executed by the DCP that each data contribution goes through beginning with first notification of a dataset (typically contact with the data generator or identification of data in a data archive or in scientific literature) all the way through data analysis by the DCP and continued through usability by data consumers
+
+## Objectives
+
+The objective of the Data Operations is to connect the scientific community to tools and resources from the DCP which maximize the value and use of HCA data such as data, metadata, analysis tools, and data access features. Data Operations aims to maintain a reliable and efficient data lifecycle and shepherds data through the data lifecycle.
+
+## In-scope
+
+- Data operations
```

Comment:
The data operations objectives below this comment, are bold (I like the ambition!), but there is a very significant risk that large chunks of the DCP requirements analysis and testing will be siloed into the data operations team in this model. I think you need to outline how the data operations team will practically accomplish these objectives. For example, you could add objectives concerned with the establishment of practices to  communicate requirements, and objectives to monitor and refine these procedures to ensure communication is effective. The major risk I see is that this will disempower and/or disincentivize teams and individuals from themselves focusing on user needs and creating proper testing procedures ("the data ops team does that, not us"), rather than supplement teams with an operational, data focused view. How do you plan to offset these risk?

Reply:
We view our means of communicating and specifying requirements as simply fitting in to the existing DCP practices, not something that needed to be explicitly stated in the charter.

We certainly want to avoid disempowering others, so suggestions are welcomed to prevent that. (also, is it bad if every charter includes "focus on user needs"?)

Reply:
Certainly it is a good thing if every charter focuses on user needs (although even better would be if this didn't even need stating!).

To pick just one specific example, the metadata schema charter says the following is in scope:
 
> Extending the schema to meet the needs of data contributors and data consumers based on the feedback they provide

By my reading, this duplicates (or at least considerably overlaps with) the metadata and standards parts of this point from this charter:

> Engage with all DCP components to identify deliverables, including data, pipelines, metadata, standards, and user features, which maximize the value and consistency of HCA data

Do you agree this is an overlap? If so, my issue is about a diffusion of responsibility, creating (at best) an additional communication overhead and at worst a situation where the DCP team provides contradictory feedback to users and stakeholders. 

You could offset this here by (for example) chartering to provide services to route user feedback to the right places, alongside the more general clauses. But I do think it's tricky, because I would hate to water down the bold intent here to provide a DCP wide, data focused view over requirements

Reply:
I see the Metadata objective as 'build & adjust the schema so that it meets the needs of contributors and consumers'
I see the DataOps objective as 'gain knowledge about Metadata's process and output' and then our objective to connect products to the community is 'make sure the community is seeing how the schema is adjusting to fit their needs' which may be something in a Snapshot that says "here's the current schema version & here's how it has changed since the last Snapshot"

Further down, we have an objective (that I just reworked slightly) to highlight how we'll route user feedback to Metadata
>Compile feedback gained from consumers through discussions at scientific conferences and ZenDesk queries and comments. Report feedback to the DCP, at large, and the specific components pertaining to the feedback, in order to facilitate system improvement

Reply:
> "here's the current schema version & here's how it has changed since the last Snapshot"

Big +1. Data Ops can be hugely valuable in helping to compile this sort of "~release~ snapshot notes"

> "Compile feedback gained from consumers through discussions at scientific conferences and ZenDesk queries and comments. Report feedback to the DCP, at large, and the specific components pertaining to the feedback, in order to facilitate system improvement"

This I'm more worried about. Specifically for metadata, trying to build a community standard with these sorts of requirements handoffs is impossible, in my experience. It's absolutely critical to assemble working groups with all stakeholders in the room. I'd see data ops playing much more of a matchmaker role here ("If you're interested in the metadata standard you should join the metadata working group, here's how to get involved..."). And indeed, data ops themselves should be a key stakeholder.

This **isn't** to say that everyone shouldn't have a bit more knowledge all round - they should - but requirements need to be gathered with the correct group of people around the table. "Correct" for me means, both a quorum of stakeholders, and those who will do the work to address any challenges. This particular challenge I see as a general problem, rather than something specific to metadata.

Reply:
DataOps will be going to scientific meetings frequently to learn how the community is using HCA data, or how they would like to. I could be wrong, but I have not heard this as a significant portion for any other component except maybe wranglers, but they go to connect with contributors.
If we hear that someone can't find how many reads were mapped to each cell or that they don't know what a certain field means from project-to-project, we don't expect them to join a working group in order to be heard. We're going to help them out, collect those notes, and relay them back to UX (to see if they've collected similar or contradictory information, or have ideas), and the metadata team, and whatever other component we see involved in a possible solution.
DataOps is not going to be super formal in their approach to getting user feedback, because that's not how most users will want to give feedback. We're going to listen to talks, check out posters, ask lots of questions, listen, and just get a feel for how the HCA is connecting to the community.

Reply:
Clarified how DataOps will connect the DCP to community feedback in be6695fa90f0dde421583e9eb3ff419776080658

Reply:
Thanks - this definitely satisfies my concerns about the risk of multiple feedback handoffs, thanks